SurfaceToolbox/SurfaceToolbox.py

```diff
@@ -503,7 +503,7 @@ def transform(inputModel, outputModel, scaleX=1.0, scaleY=1.0, scaleZ=1.0, trans
   def translateCenterToOrigin(inputModel, outputModel):
     """Translate center of the mesh bounding box to the origin.
     """
-    bounds = inputModel.GetMesh().GetBounds()
+    bounds = inputModel.GetMesh().GetRASBounds()

```

Comment:
If the output mesh is under a transform that has rotation or scaling then this will not center the model. I think it is cleaner to keep all transform inputs and outputs in the node's coordinate system (as it is now), but if you want to have an option to translate, rotate, or center in world coordinate system then add a "Transform in RAS coordinate system" checkbox. If unchecked (default) then keep the current implementation. If checked then you need to compute how to modify the model points in the node's coordinate system that achieves the appropriate transformation in the world coordinate system.